shapes/neurosciencegraph/commons/brainlocation/schema.json

```diff
@@ -29,9 +29,20 @@
         {
           "path": "nsg:layer",
           "name": "Brain region layer",
-          "description": "Brain region region (e.g. layer of the neocortex)",
+          "description": "Brain region layer (e.g. layer of the neocortex)",
           "node": "https://neuroshapes.org/commons/typedlabeledontologyterm/shapes/BrainRegionOntologyTermShape"
         },
+        {
+          "path": "nsg:hemisphere",
+          "name": "Brain hemisphere",
```

Comment:
nsg:brainHemisphere may be ?

Reply:
Good idea. Done.

Reply:
Thanks for the feedback. I have updated t he property to "brainHemisphere"